web/ui/README.md

```diff
@@ -5,10 +5,8 @@ using the vfsgen library (c.f. Makefile).
 During development it is more convenient to always use the files on disk to
 directly see changes without recompiling.
 To make this work, remove the `builtinassets` build tag in the `flags` entry
-in `.promu.yml`, and then `make build`.
+in `.promu.yml`, and then `make build` (or build Prometheus using
+`go build ./cmd/prometheus`).
 
 This will serve all files from your local filesystem.
 This is for development purposes only.
-
-After making changes to any file, run `make assets` before committing to update
```

Comment:
this step isn't needed anymore?

Reply:
Yeah, as built assets are no longer checked in, the `assets` target (which includes building the React bundle) is now run automatically when you run `make build`.